src/lua/httpc.lua

```diff
@@ -291,6 +298,31 @@ local function encode_body(body, content_type)
     return raw_body, content_type
 end
 
+local function decode_body(response)
+    if response.body == nil then
+        error('Unable to decode body: body is empty', 2)
+    end
+    local headers = response.headers or {}
+    local content_type = header_value_wo_parameters(headers['content-type'])
+    local default_decoder = function(raw_body, _)
```

Comment:
Maybe use the new unused var naming (`_content_type`) or just drop it? This way you can remind of the second parameter constructively.